nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutDatabaseRecord.java

```diff
@@ -301,13 +301,23 @@
             .name("put-db-record-max-batch-size")
             .displayName("Maximum Batch Size")
             .description("Specifies maximum batch size for INSERT and UPDATE statements. This parameter has no effect for other statements specified in 'Statement Type'."
-                            + " Zero means the batch size is not limited.")
+                    + " Zero means the batch size is not limited.")
             .defaultValue("0")
             .required(false)
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
             .build();
 
+    static final PropertyDescriptor UPSERT_DO_NOTHING = new PropertyDescriptor.Builder()
```

Comment:
This feels odd to me. A property named "Upsert Do Nothing" I feel is confusing and misleading. It sounds like any upsert should be ignored and not acted upon. And in this case, it's not really upserts that are being ignored, but conflicting Inserts. What makes more sense to me is to add another option for the Statement Type: "INSERT_IGNORE" or something to that effect. This would be consistent with how it is done with PutKudu, also.

Reply:
Thanks for your reply and sorry for making that misleading. "INSERT_IGNORE" is a good idea. I will try it later :D